velox/functions/CoverageUtil.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include <unordered_set>
+
+void printCoverageMapForAll(const std::string& domain = "");
```

Comment:
Nice small header file. Thank you for cleaning it up. Let's document the remaining functions. In particular, let's clarify what "domain" is and explain that these functions expect to find certain files at certain paths.

Reply:
Thanks, I added some documents for the remaining functions.